src/BPSecLib_Public.h

```diff
@@ -118,6 +118,28 @@ typedef enum
     BSL_BUNDLECRCTYPE_32 = 2,
 } BSL_BundleCRCType_e;
 
+typedef enum
+{
+    BSL_TLM_BUNDLE_INSPECTED_COUNT = 0,
+    BSL_TLM_BUNDLE_INSPECTED_BYTES,
+    BSL_TLM_ASB_DECODE_COUNT,
+    BSL_TLM_ASB_DECODE_BYTES,
+    BSL_TLM_ASB_ENCODE_COUNT,
+    BSL_TLM_ASB_ENCODE_BYTES,
+    BSL_TLM_SECOP_SOURCE_COUNT,
+    BSL_TLM_SECOP_VERIFIER_COUNT,
+    BSL_TLM_SECOP_ACCEPTOR_COUNT,
+    BSL_TLM_SECOP_FAIL_COUNT,
+    BSL_TLM_TOTAL_COUNT
+} BSL_TlmCounterIndex_e;
+
+typedef struct BSL_TlmCounters_s
+{
+    uint64_t counters[BSL_TLM_TOTAL_COUNT + 1];
+} BSL_TlmCounters_t;
+
+int BSL_LibCtx_GetTlmCounters(const BSL_LibCtx_t *lib, BSL_TlmCounters_t *tlm);
```

Comment:
While this does work and is easy for the single-thread, it may be better to change this from "Get" to "Accumulate" so that the `tlm` parameter is added-to from the local counters with a static initializer for zero-values like
```c
#define BSL_TLM_COUNTERS_ZERO (BSL_TlmCounters_t){ 0 }
```